runtime/vm/MHInterpreter.hpp

```diff
@@ -383,81 +383,6 @@ class VM_MHInterpreter
 		}
 		return methodHandle;
 	}
-
-#ifdef J9VM_OPT_PANAMA
```

Comment:
Line 44 which includes `FFITypeHelpers.hpp` should be deleted (that file should be removed).